dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/EventController.java

```diff
@@ -652,6 +654,14 @@ public void getCsvEvents(
             outputStream = new GZIPOutputStream( outputStream );
             response.setContentType( "application/csv+gzip" );
         }
+        else if ( ContextUtils.isAcceptCsvZip( request ) )
+        {
```

Comment:
I get a bit of a "centralize this" feeling but it might not be worth it.

Reply:
No, I don't in this case it's not so important. This code is on the way out, and as you see the tracker team has rewritten it to not use the message converters anymore.